chains/evm/contracts/pools/USDC/HybridLockReleaseUSDCTokenPool.sol

```diff
@@ -15,7 +14,7 @@ import {IERC20} from
 import {SafeERC20} from
   "@chainlink/contracts/src/v0.8/vendor/openzeppelin-solidity/v4.8.3/contracts/token/ERC20/utils/SafeERC20.sol";
 import {EnumerableSet} from
-  "@chainlink/contracts/src/v0.8/vendor/openzeppelin-solidity/v4.8.3/contracts/utils/structs/EnumerableSet.sol";
+  "@chainlink/contracts/src/v0.8/vendor/openzeppelin-solidity/v5.0.2/contracts/utils/structs/EnumerableSet.sol";
```

Comment:
All non-token OZ imports are 5.x. We cannot (easily) migrate the token related ones to 5.x because we use 4.x token contracts from the `chainlink-evm` repo. This is fine, these changes just make it consistent that we use 5.x for non-token stuff.